### PR TITLE
AF-1787 - Integrate AppFormer.js on Business Central

### PR DESCRIPTION
**Pull request by tiagobento**
On this PR:

- Configuration of new AppFormer.js component Spaces Screen

Related PRs:
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/936
https://github.com/kiegroup/appformer/pull/662
https://github.com/kiegroup/kie-wb-common/pull/2526
https://github.com/kiegroup/drools-wb/pull/1090
https://github.com/kiegroup/jbpm-wb/pull/1314
https://github.com/kiegroup/optaplanner-wb/pull/329
https://github.com/kiegroup/kie-wb-distributions/pull/893

### CODE REVIEW

**Comment by tiagobento** [comment]
Jenkins, please retest this.

**Comment by domhanak** [comment]
@rsynek mind having a look? :)

File: optaplanner-wb-webapp/pom.xml [hunk 1]
```diff
@@ -233,6 +233,12 @@
     <dependency>
       <groupId>org.uberfire</groupId>
       <artifactId>uberfire-structure-backend</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>org.uberfire</groupId>
+          <artifactId>uberfire-servlet-security</artifactId>
```
Comment (domhanak) [inline optaplanner-wb-webapp/pom.xml:239]: Why do we exclude this?

File: optaplanner-wb-ui/pom.xml [hunk 2]
```diff
@@ -1424,4 +1434,69 @@
 
   </build>
 
+  <profiles>
+    <profile>
+      <id>not-ts-exporter-build</id>
```
Comment (rsynek) [inline optaplanner-wb-ui/pom.xml]: Would you maybe put there a comment explaining why is this configuration needed?

File: optaplanner-wb-ui/src/main/java/org/optaplanner/workbench/client/OptaPlannerWorkbenchEntryPoint.java [hunk 3]
```diff
@@ -75,6 +81,12 @@ public OptaPlannerWorkbenchEntryPoint(final Caller<AppConfigService> appConfigSe
         this.adminPage = adminPage;
         this.translationService = translationService;
         this.placeManager = placeManager;
+        this.appFormerJsBridge = appFormerJsBridge;
+    }
+
+    @PostConstruct
+    protected void setup() {
+        this.appFormerJsBridge.init("optaplanner-wb/org.optaplanner.workbench.OptaPlannerWorkbench");
```
Comment (rsynek) [inline optaplanner-wb-ui/src/main/java/org/optaplanner/workbench/client/OptaPlannerWorkbenchEntryPoint.java:89]: If this is a unique identifier of this particular workbench, turning it into a well-named constant will make it clear.

**Review by rsynek — APPROVED** [review]
Generally looks good, two soft suggestions regarding clarity of the code.

**Review by domhanak — APPROVED** [review]

File: optaplanner-wb-ui/pom.xml (hunk 2 above)
Comment (tiagobento) [reply optaplanner-wb-ui/pom.xml]: This was removed.

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**